mimium-lang/src/compiler/parser.rs

```diff
@@ -116,11 +116,13 @@ where
     prec.clone()
         .then(op.then(prec).repeated())
         .foldl(move |x, ((op, opspan), y)| {
-            Expr::Apply(
-                Expr::Var(op.get_associated_fn_name()).into_id(opspan),
-                vec![x, y],
-            )
-            .into_id(x.to_span().start..y.to_span().end)
+            let arg = match op {
+                // A@B is a syntactic sugar of _mimium_schedule_at(B, A)
+                Op::At => vec![y, x],
```

Comment:
Is it better to fix the order of the argument of `_mimium_schedule_at` itself for the simplicity of here?

Reply:
Thanks, I was wondering about this, too. Are there a chance that `@` eventually accept function call with arguments like described in [the v1 document](https://mimium.org/ja/docs/users-guide/language-specification/grammar-rule/#%E6%BC%94%E7%AE%97%E5%AD%90%E3%81%AB%E3%82%88%E3%82%8B%E9%81%85%E5%BB%B6%E5%AE%9F%E8%A1%8C)?

```
println(100)@0
```

If so, is `_mimium_schedule_at(0, println, 100)` better than `_mimium_schedule_at(println, 0, 100)` because the function and argument are in order?